reflex/event.py

```diff
@@ -459,7 +466,7 @@ def set_focus(ref: str) -> EventSpec:
     return server_side(
         "_set_focus",
         get_fn_signature(set_focus),
-        ref=Var.create_safe(format.format_ref(ref)),
+        ref=Var.create_safe(format.format_ref(ref), _var_is_string=True),
```

Comment:
Why are refs strings now?

Reply:
that's what causes them to get rendered with backticks now, see example above.